### ISSUE TITLE
ZigBee pairing fails on "This device is already paired."

### ISSUE THREAD

**Issue by TedTolboom** [athombv/node-homey-meshdriver#8]
1.5.0 RC8 - latest meshdriver

While trying to pair the Gira Light Link remote, the device is recognized:
"Tue, 22 Aug 2017 17:00:01 GMT zigbee-shepherd:request identified device (manufacturer:  Insta , product:  Remote)" LOG reading from @RobinBol 

but pairing is terminated with message "This device is already paired. Try removing it from your devices or via Settings -> ZigBee."
<img width="596" alt="schermafbeelding 2017-08-22 om 19 01 13" src="https://user-images.githubusercontent.com/17163431/29586843-4976e3be-878c-11e7-93aa-96dedb655074.png">

the Zigbee map shows an Unknown device without any Device information:
<img width="724" alt="schermafbeelding 2017-08-22 om 19 01 55" src="https://user-images.githubusercontent.com/17163431/29586853-5883d7ae-878c-11e7-96e7-bc232672ece3.png">

Removing the Unknown device, resetting the Light Link, resetting the Zigbee network has no effect.
Trying pairing with or without app installed / Ikea Bulb turned on, no effect

LOG: F07D0A3233

**Comment by TedTolboom** [comment]
when trying to remove (using remove button above), I get the following error:
<img width="398" alt="schermafbeelding 2017-08-24 om 21 53 13" src="https://user-images.githubusercontent.com/17163431/29685861-0b174d20-8917-11e7-99c3-adbc1c6fce36.png">

**Comment by TedTolboom** [comment]
Checked the communication between the Gira LightLink and Homey...

As far as I can see the right identification is provided and included into the app... 
<img width="1112" alt="schermafbeelding 2017-08-31 om 23 43 33" src="https://user-images.githubusercontent.com/17163431/29946888-63ab7da4-8ea7-11e7-96a0-9714a13f9242.png">

but at a certain moment the GLL receives a LEAVE command from Homey:

See attached Wireshark dump of the inclusion:
[Gira_LighLink inclusion.pcapng.zip](https://github.com/athombv/node-homey-meshdriver/files/1268617/Gira_LighLink.inclusion.pcapng.zip)

Let me know if you need further information

**Comment by TedTolboom** [comment]
Tried to include the Gira Remote on my brand new, never touched 2nd Homey....

Installed only the Gira app, and tried to include the Gira Remote...
Same error: this device is already paired... ?

(ultra clean) LOG: CECA20FE81

In addition, I see several other comments on the forum about similar issues...

**Comment by TedTolboom** [comment]
@RobinBol also checked on 1.5.3 and 1.2.0 meshdriver... 
same result... any ideas how to correct?

**Comment by RobinBol** [comment]
As communicated with Ted, and for the others; this is most likely fixed in the next Homey firmware update, if not please let me know!

**Comment by RobinBol** [comment]
@TedTolboom can this issue be closed or are you still encountering this error message?

**Comment by TedTolboom** [comment]
@RobinBol I'm still awaiting the release of 1.5.4... 
which should have THE magic shepherd update which should fix this issue and bring some other nice goodies...

Still the same error on 1.5.3 and meshdriver 1.2.4..

**Comment by RobinBol** [comment]
Ah, I am sorry.. I'll have to be a bit more patient then:)

**Comment by morgendagen** [comment]
Could this be the same issue I am facing?

I have successfully added a Hue dimmer switch, a couple of lamps, and a motion sensor, using my own fork of the com.hue.zigbee app, adding motion sensor support. Just now I tried adding some Hue ambiance GU10 bulbs. After trying to pair approx. 10 times getting the "This device is already paired" message, I succeeded in adding one bulb, but am unable to add any more.
